### DISCUSSION TITLE
채팅 드래그 & 복사기능

### DISCUSSION THREAD

**Discussion by act-lamos** [metamong-friends/metamong-friends.github.io#64] [제안하기]
채팅창 메시지를 드래그 해서 복사 할 수 있으면 좋을 것 같습니다.

**Comment by shlee303** [comment]
@act-lamos 많이 기다리셨습니다! 이번 릴리즈에 채팅 복사 하실 수 있도록 개선 반영 됐습니다. 항상 감사합니다 Lamos!
